app/forms/idv/api_image_upload_form.rb

```diff
@@ -192,8 +201,13 @@ def update_analytics(client_response)
       )
     end
 
+    def acuant_sdk_capture?
+      image_metadata.dig(:front, :source) == 'acuant' &&
+        image_metadata.dig(:back, :source) == 'acuant'
+    end
+
     def image_metadata
-      params.permit(:front_image_metadata, :back_image_metadata).
+      @_image_metadata ||= params.permit(:front_image_metadata, :back_image_metadata).
```

Comment:
I'd avoid the leading underscore in the ivar name, I don't think it gets us anything, since nothing clashes with it?

```suggestion
      @image_metadata ||= params.permit(:front_image_metadata, :back_image_metadata).
```

Reply:
> I'd avoid the leading underscore in the ivar name, I don't think it gets us anything, since nothing clashes with it?

This is one I've felt uncertain on, since I've seen a pretty even split throughout the codebase. I must have been feeling the leading underscore today. I could see some value in communicating it as a "private" variable never meant to be referenced, even internally in the class.

I really don't feel strongly at all though. I'll drop it. And maybe I / we should follow-up to make this more consistent throughout?

```
ag "def (\w+)\s+@_?\1 \|\|="
```

Reply:
🤷 I don't feel that strongly, but it would be nice to have a follow-up ticket to make it consistent

Reply:
Updated to remove leading underscore in e025786, and created a follow-up ticket for cleanup at LG-4866.